### ISSUE TITLE
Similtu filemanager doesn't work

### ISSUE THREAD

**Issue by alibaraneser** [UniSharp/laravel-filemanager#622]
Please check the following instructions before submitting a bug :
 * Make sure you are using the latest version.
 * Make sure you read [installation](http://unisharp.github.io/laravel-filemanager/installation), [integration](http://unisharp.github.io/laravel-filemanager/integration), and [upgrade](http://unisharp.github.io/laravel-filemanager/upgrade) document.

And provide the followings :
 * Operating system :
 * Laravel version :
 * Package version :
 * Steps to reproduce your issue :
 * Screenshots of browser console :

**Comment by g0110280** [comment]
Can you post some screenshots here? The link you provided only shows home page.

**Comment by alibaraneser** [comment]
what exactly do you want,  kind of photos ?

**Comment by g0110280** [comment]
Screenshots of the error you mentioned. The link you provided, which redirected me to https://www.photobox.co.uk/ , shown me no error.

**Comment by alibaraneser** [comment]
https://ibb.co/hJAXpo now ? :)

**Comment by g0110280** [comment]
I see now. Maybe some steps are missing after you run `composer update`? Try running `php artisan vendor:publish --tags=lfm_public --force`.

**Comment by alibaraneser** [comment]
Yeah, I did not publish vendors when I pulled out with git. Now it worked thx. :+1:
